### PR TITLE
[DOCS] Add ml-cpp PRs to 7.9.0 release notes

### PR DESCRIPTION
**Pull request by lcawl**
This PR copies the content from https://raw.githubusercontent.com/elastic/ml-cpp/7.9/docs/CHANGELOG.asciidoc into the Elasticsearch release notes (https://www.elastic.co/guide/en/elasticsearch/reference/7.9/release-notes-7.9.0.html)

### Preview 

https://elasticsearch_60689.docs-preview.app.elstc.co/guide/en/elasticsearch/reference/7.x/release-notes-7.9.0.html

### CODE REVIEW

File: docs/reference/release-notes/7.9.asciidoc [hunk 1]
```diff
@@ -69,6 +69,7 @@ Geo::
 Machine Learning::
 * Add update data frame analytics jobs API {es-pull}58302[#58302] (issue: {es-issue}45720[#45720])
 * Introduce model_plot_config.annotations_enabled setting for anomaly detection jobs {es-pull}57539[#57539] (issue: {es-issue}55781[#55781])
+* Report significant changes to anomaly detection models in annotations of the results {ml-pull}1247[#1247], {pull}56342[#56342], {pull}56417[#56417], {pull}57144[#57144], {pull}57278[#57278], {pull}57539[#57539]
```
Comment (lcawl) [inline docs/reference/release-notes/7.9.asciidoc:72]:
```suggestion
* Report significant changes to anomaly detection models in annotations of the results {ml-pull}1247[#1247], {ml-pull}56342[#56342], {ml-pull}56417[#56417], {ml-pull}57144[#57144], {ml-pull}57278[#57278], {ml-pull}57539[#57539]
```

File: docs/reference/release-notes/7.9.asciidoc [hunk 2]
```diff
@@ -184,6 +185,18 @@ Machine Learning::
 * Delete auto-generated annotations when model snapshot is reverted {es-pull}58240[#58240] (issue: {es-issue}57982[#57982])
 * Delete expired data by job {es-pull}57337[#57337]
 * Introduce Annotation.event field {es-pull}57144[#57144] (issue: {es-issue}55781[#55781])
+* Add support for larger forecasts in memory via max_model_memory setting {ml-pull}1238[#1238],{pull}57254[#57254]
```
Comment (lcawl) [inline docs/reference/release-notes/7.9.asciidoc:188]:
```suggestion
* Add support for larger forecasts in memory via max_model_memory setting {ml-pull}1238[#1238],{ml-pull}57254[#57254]
```

File: docs/reference/release-notes/7.9.asciidoc [hunk 3]
```diff
@@ -184,6 +185,18 @@ Machine Learning::
...
+* Don't lose precision when saving model state {ml-pull}1274[#1274]
+* Parallelize the feature importance calculation for classification and regression over trees {ml-pull}1277[#1277]
+* Add an option to do categorization independently for each partition {ml-pull}1293[#1293], {ml-pull}1318[#1318], {ml-pull}1356[#1356], {pull}57683[#57683]
```
Comment (lcawl) [inline docs/reference/release-notes/7.9.asciidoc:191]:
```suggestion
* Add an option to do categorization independently for each partition {ml-pull}1293[#1293], {ml-pull}1318[#1318], {ml-pull}1356[#1356], {ml-pull}57683[#57683]
```

File: docs/reference/release-notes/7.9.asciidoc [hunk 4]
```diff
@@ -69,6 +69,7 @@ Geo::
 Machine Learning::
 * Add update data frame analytics jobs API {es-pull}58302[#58302] (issue: {es-issue}45720[#45720])
 * Introduce model_plot_config.annotations_enabled setting for anomaly detection jobs {es-pull}57539[#57539] (issue: {es-issue}55781[#55781])
+* Report significant changes to anomaly detection models in annotations of the results {ml-pull}1247[#1247], {ml-pull}56342[#56342], {ml-pull}56417[#56417], {ml-pull}57144[#57144], {ml-pull}57278[#57278], {ml-pull}57539[#57539]
```
Comment (droberts195) [inline docs/reference/release-notes/7.9.asciidoc:72]: Only 1247 is an `ml-pull`.  The ones over 50000 should be `es-pull`.

```suggestion
* Report significant changes to anomaly detection models in annotations of the results {ml-pull}1247[#1247], {es-pull}56342[#56342], {es-pull}56417[#56417], {es-pull}57144[#57144], {es-pull}57278[#57278], {es-pull}57539[#57539]
```

File: docs/reference/release-notes/7.9.asciidoc [hunk 5]
```diff
@@ -184,6 +185,18 @@ Machine Learning::
 * Delete auto-generated annotations when model snapshot is reverted {es-pull}58240[#58240] (issue: {es-issue}57982[#57982])
 * Delete expired data by job {es-pull}57337[#57337]
 * Introduce Annotation.event field {es-pull}57144[#57144] (issue: {es-issue}55781[#55781])
+* Add support for larger forecasts in memory via max_model_memory setting {ml-pull}1238[#1238], {ml-pull}57254[#57254]
```
Comment (droberts195) [inline docs/reference/release-notes/7.9.asciidoc:188]:
```suggestion
* Add support for larger forecasts in memory via max_model_memory setting {ml-pull}1238[#1238], {es-pull}57254[#57254]
```

File: docs/reference/release-notes/7.9.asciidoc [hunk 6]
```diff
@@ -184,6 +185,18 @@ Machine Learning::
...
+* Don't lose precision when saving model state {ml-pull}1274[#1274]
+* Parallelize the feature importance calculation for classification and regression over trees {ml-pull}1277[#1277]
+* Add an option to do categorization independently for each partition {ml-pull}1293[#1293], {ml-pull}1318[#1318], {ml-pull}1356[#1356], {ml-pull}57683[#57683]
```
Comment (droberts195) [inline docs/reference/release-notes/7.9.asciidoc:191]:
```suggestion
* Add an option to do categorization independently for each partition {ml-pull}1293[#1293], {ml-pull}1318[#1318], {ml-pull}1356[#1356], {es-pull}57683[#57683]
```

File: docs/reference/release-notes/7.9.asciidoc (hunk 5 above)
Comment (lcawl) [reply docs/reference/release-notes/7.9.asciidoc:188]: Good catch, thanks!

**Review by droberts195 — APPROVED** [review]
LGTM

**Pull request merged**